apps/sms/test/marionette/conversation_test.js

```diff
@@ -491,4 +503,101 @@ marionette('Conversation Panel Tests', function() {
       });
     });
   });
+
+  suite('Add recipients from thread', function() {
+    var conversationView;
+    setup(function() {
+      var thread = ThreadGenerator.generate({
+        numberOfMessages: 1
+      });
+      storage.setMessagesStorage([thread], ThreadGenerator.uniqueMessageId);
```

Comment:
It seems I don't have any failure... But how do you check for intermittent failure on taskcluster? Does the automatic retry possibly hide them from the results? Do you know which Gij number it is by heart and you check manually?